PrivateCollections/Germany/Neukirchen005.xml

```diff
@@ -141,98 +141,171 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                                <ab type="ruling" subtype="pattern">Ruling pattern: 1-1-11/0/0/C</ab>
+                                <note corresp="#leafdim">Data taken from fol. 161r.</note>
+
+                                <dimensions unit="mm" xml:id="textarea2">
+                                   <height>94</height>
+                                   <width>130</width>
+                                </dimensions>
+
+                                <dimensions type="margin" unit="mm" xml:id="margin2">
+                                   <dim type="top">17</dim>
+                                   <dim type="bottom">38</dim>
+                                   <dim type="right">11</dim>
+                                   <dim type="left">11</dim>
+                                </dimensions>
+                              </layout>
                      </layoutDesc>
                   </objectDesc>
                   <handDesc>
                      <handNote xml:id="h1" script="Ethiopic">
                         <seg type="script"/>
                         <seg type="ink">Black, red</seg>
-                        <seg type="rubrication">First one or two lines in the beginning of the text and text sections; numerals; punctuation marks;
+                        <seg type="rubrication">Titles and incipits of the psalms and other texts; numbers
+                          of psalms and other texts; punctuation marks;
                         name of God, Mary, and angels; single letters and words.</seg>
                         <desc/>
                      </handNote>
                   </handDesc>
 
                   <additions>
                     <list>
+                      <item xml:id="a1">
+                        <locus target="#159v"/>
+                        <desc type="MagicText" cert="low"/>
+                        <note>Crudely written, to the most part erased.
+                          Written in the main hand.</note>
+                      </item>
+
+                      <item xml:id="a2">
+                        <locus target="#178v"/>
+                        <desc type="GuestText">Supplications, probably from
+                          <ref type="work" corresp="LIT1931Mashaf">Maṣḥafa gənzat</ref>.</desc>
+                        <note>Crudely written, partly unreadable. In the text, a patriarch Yohannes
+                      and an Ethiopian metropolitan are mentioned.</note>
+                      </item>
+
                       <item xml:id="e1">
-                        <locus target="#1v"></locus>
-                        <desc> A crude unfinished drawing of an animal.</desc>
+                        <locus from="1r" to="2v"/>
+                        <desc type="GuestText">Possibly excerpts from a
+                          <ref type="work" corresp="LIT4018Mashafa">Maṣḥafa gəṣṣāwe</ref>.</desc>
+                        <note>A re-utilized double leaf originating from
```

Comment:
according to Gumbert this is part of binding not of MS; if part of ms then it is a msPart

Reply:
The proble is that Gumbert did not deal much with Ethiopic manuscripts; a big part of his experience originates from mss where "paste-downs" are used extensively that are made of protective leaves indeed a part of binding, not of the text block. We rarely have this in Ethiopic manuscripts where the position of the protective quires is much more close to the textblock in the bulk of the mss (it is very clear only in the mss in the European libraries which were rebound). These two leaves are not on the same level as the other ms-parts (in this case only one). It is an accidental later insertion, the text is not considered for reading at all (I will add this). If we encode it as ms-part, it will also obscure the entire "programm" of the ms-makers (which has nothing to do with Mashafa gessawe).

Reply:
But I arranged the hands in the physical sequence as they appear in the volume.

Reply:
good